docs/flow/modin/experimental/pandas.rst

```diff
@@ -13,4 +13,4 @@ Experimental API Reference
 .. autofunction:: read_csv_glob
 .. autofunction:: read_custom_text
 .. autofunction:: read_pickle_distributed
-.. automethod:: modin.experimental.pandas.DataFrame.to_pickle_distributed
+.. automethod:: modin.pandas.DataFrame.modin::to_pickle_distributed
```

Comment:
`As a hint to autodoc extension, you can put a :: separator in between module name and object name to let autodoc know the correct module name if it is ambiguous.` from https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-automethod